drivers/pinctrl/pinctrl_lpc_iocon.c

```diff
@@ -11,22 +11,40 @@
 #include <fsl_clock.h>
 #endif
 
+/* IOCON register addresses. */
+static uint32_t *iocon[] = {
```

Comment:
```suggestion
static uint32_t *volatile iocon[] = {
```
I think the pointers in this array need to be declared as volatile still

Reply:
Updated.